core/test/integration/synchronization/synchronization_tests.cpp

```diff
@@ -269,6 +269,120 @@ TEST_F(BitcoinLikeWalletSynchronization, TestNetSynchronization) {
     }
 }
 
+TEST_F(BitcoinLikeWalletSynchronization, MultipleSynchronization) {
+    auto pool = newDefaultPool();
+    {
+        auto wallet = wait(pool->createWallet("e847815f-488a-4301-b67c-378a5e9c8a61", "bitcoin",
+                                              api::DynamicObject::newInstance()));
+        std::vector<std::string> xpubs = {
+                P2PKH_MEDIUM_XPUB_INFO.extendedKeys[0]
+        };
+        std::vector<std::shared_ptr<BitcoinLikeAccount>> accounts;
+        for (auto x : xpubs) {
+            auto nextIndex = wait(wallet->getNextAccountIndex());
+            auto info = wait(wallet->getNextExtendedKeyAccountCreationInfo());
+            info.extendedKeys.push_back(x);
+            auto account = createBitcoinLikeAccount(wallet, nextIndex, info);
+            accounts.push_back(account);
+        }
+
+        std::set<std::string> emittedOperations;
+        {
+            auto receiver = make_receiver([&](const std::shared_ptr<api::Event> &event) {
+                if (event->getCode() == api::EventCode::NEW_OPERATION) {
+                    auto uid = event->getPayload()->getString(
+                            api::Account::EV_NEW_OP_UID).value();
+                    EXPECT_EQ(emittedOperations.find(uid), emittedOperations.end());
+                }
+            });
+
+            pool->getEventBus()->subscribe(dispatcher->getMainExecutionContext(), receiver);
+
+            static std::function<void (int)> syncAccount = [accounts, receiver, this](int index){
```

Comment:
The static storage isn't needed here.

```suggestion
            std::function<void (int)> syncAccount;
            
            syncAccount = [accounts, receiver, this](int index){
 
```

Reply:
As discussed f2f: it's used for convenience and more readability